release/release.go

```diff
@@ -37,41 +37,33 @@ type OS struct {
 	VersionID string `json:"version-id,omitempty"`
 }
 
+var (
+	versionSignaturePath = "/proc/version_signature"
```

Comment:
@jdstrand, I agree but this is by design. With the exception of testing mailing kernels we should clearly and loudly refuse to function in this environment.